x-pack/plugins/index_management/public/app.js

```diff
@@ -4,17 +4,26 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React from 'react';
+import React, { Component } from 'react';
 import { Switch, Route, Redirect } from 'react-router-dom';
-import { BASE_PATH } from '../common/constants';
+import { BASE_PATH, UA_APP_LOAD } from '../common/constants';
 import { IndexList } from './sections/index_list';
+import { trackUserAction } from './services';
 
-export const App = () => (
-  <div>
-    <Switch>
-      <Redirect exact from={`${BASE_PATH}`} to={`${BASE_PATH}indices`}/>
-      <Route exact path={`${BASE_PATH}indices`} component={IndexList} />
-      <Route path={`${BASE_PATH}indices/filter/:filter?`} component={IndexList}/>
-    </Switch>
-  </div>
-);
+export class App extends Component {
+  componentWillMount() {
```

Comment:
This lifecycle is deprecated (https://reactjs.org/blog/2018/03/29/react-v-16-3.html). We should use `componentDidMount()`